### PR TITLE
Projects Page Layout Adjustments

### LINKED ISSUE #98: Enhancement: Re-distribute whitespace on projects page
### Describe the Current Behavior/Feature:

Change the layout of project page to more evenly distribute whitespace.

---

### Proposed Implementation:

Adjust alignment and padding.

### PR DESCRIPTION
**Pull request by ogorman89**
## This PR:

This is just a personal preference, but what do you guys think of this adjustment to the project's page layout? If you guys like the current layout better, that's cool too. Just wanted to throw it out there. ✌️

Resolves #98 by adjusting margins and svg sizes.

---

## Screenshots (if applicable):

Before: 
Screen Shot 2024-01-23 at 10 21 51 PM

Screen Shot 2024-01-23 at 10 22 02 PM

After:

Screen Shot 2024-01-23 at 10 47 06 PM

Screen Shot 2024-01-23 at 10 47 47 PM

Screen Shot 2024-01-23 at 10 47 32 PM

---

## Future Steps/PRs Needed to Finish This Work (optional):

N/A

### CODE REVIEW

File: src/pages/Projects.jsx [hunk 1]
```diff
@@ -61,18 +62,14 @@ const ProjectTitle = ({ index, title, logo, links }) => {
           sx={{
             objectFit: 'fill',
             maxWidth: '80%',
-            height: logo === '/assets/codePdx/pass-logo.svg' ? 'auto' : '150px',
-            pt: logo != '/assets/codePdx/logo-record-sponge-oregon.svg' ? '5%' : 'none'
+            height: { xs: '55px', sm: '100px' }, //55 logo === '/assets/codePdx/pass-logo.svg' ? 'auto' : '150px'
+            // pt: logo != '/assets/codePdx/logo-record-sponge-oregon.svg' ? '5%' : 'none'
```
Comment (andycwilliams) [inline src/pages/Projects.jsx]: If we go with these changes, should remove the commented-out code

**Review by andycwilliams — COMMENTED** [review]
Yeah looks a little nicer centered this way. We might want more editing later on, such as making the CODE PDX logo a little larger. But I'm fine with these changes. Or none of them.

**Review by Jared-Krajewski — APPROVED** [review]
🚀

File: src/pages/Projects.jsx (hunk 1 above)
Comment (ogorman89) [reply src/pages/Projects.jsx]: I removed these comments and merged.

**Pull request merged**